db/seeds.rb

```diff
@@ -1,12 +1,74 @@
 # frozen_string_literal: true
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
-user = User.create(email: 'test@test.com', password: '123456' )
+user1 = User.create(email: 'test1@test.com', password: '123456' )
+user2 = User.create(email: 'test2@test.com', password: '123456' )
+user3 = User.create(email: 'test3@test.com', password: '123456' )
+user4 = User.create(email: 'test4@test.com', password: '123456' )
 
-12.times do |el|
+8.times do |el|
   CuriosityCard.create(title: 'Hello world',
                        content: "this is my curiosity No.#{el}",
-                       user_id: User.first.id)
+                       user_id: user1.id)
```

Comment:
Is it ok for user1 to have 8 curiosity card with the same title and content? 👀 
Same with the rest of records down below. It's doesn't make a lot of sense

Reply:
you're right, thanks :D